Sources/TuistDependencies/SwiftPackageManager/Utils/PackageInfoMapper.swift

```diff
@@ -291,6 +291,34 @@ public final class PackageInfoMapper: PackageInfoMapping {
         packageToProject: [String: AbsolutePath],
         swiftToolsVersion: TSCUtility.Version?
     ) throws -> ProjectDescription.Project? {
+        // Hardcoded mapping for some well known libraries, until the logic can handle those properly
+        let productTypes = productTypes.merging(
+            [
+                // Force dynamic frameworks
+                "RxSwift": .framework // https://github.com/ReactiveX/RxSwift
+            ],
+            uniquingKeysWith: { userDefined, _ in userDefined }
```

Comment:
Just to be sure, this should take user configs and override with any defaults? So if someone made rxswift static it'll stay static?

Reply:
exactly :sisi: